CIP-????/README.md

```diff
@@ -0,0 +1,311 @@
+    - https://github.com/cardano-foundation/CIPs/pull/1117
+Created: 2025-11-29
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP introduces the **Compensatory Ledger Mechanism (CLM)**—a deterministic, protocol-level compensation system for governance actors participating in Cardano governance actions. A portion of each governance action’s deposit is automatically distributed to Constitutional Committee (CC) members, Delegated Representatives (DReps), and Stake Pool Operators (SPOs) who cast a valid vote.
+
+This mechanism requires **no smart contracts**, **no Treasury withdrawals**, and **no manual triggers**, operating fully within Cardano’s ledger rules in a manner analogous to staking rewards. It creates an incentive-aligned, sustainable, and decentralized compensation model that reflects the real cost of governance participation.
+
+---
+
+## Motivation: why is this CIP necessary?
+
+Cardano’s decentralized governance framework requires persistent engagement from CC members, DReps, and SPOs. These actors must analyze governance actions, evaluate technical and economic implications, and cast informed votes. This work incurs ongoing time, expertise, and operational costs. Without a reliable compensation model, several risks emerge:
+
+### Governance Fatigue
+Uncompensated governance labor leads to declining participation over time.
+
+### Centralization Risks
+Only large, well-funded entities can consistently afford to participate, reducing diversity and representation.
+
+### Low Governance Participation / Quorum Instability
+Without compensation, new and smaller actors lack incentives to engage, harming governance quality.
+
+### Misaligned Incentives
+Proposers incur no cost commensurate with the burden placed on reviewers and voters.
+
+### Need for a Deterministic, Automated Mechanism
+Any off-chain or trust-based compensation scheme introduces friction and reduces security.
+
+The CLM addresses these issues by:
+- Making governance participation economically feasible  
+- Ensuring compensation flows only to active participants  
+- Funding compensation from proposer deposits, aligning costs with governance load  
+- Executing payments entirely through the ledger for maximum reliability  
+
+---
+
+## Specification
+
+### New Protocol Parameters
+
+The following fields are added to `PParams`:
+
+| Parameter | Type | Description |
+|----------|------|-------------|
+| `ccCompensationRate` | UnitInterval | Portion of `govDeposit` allocated to CC. |
+| `drepCompensationRate` | UnitInterval | Portion allocated to DReps. |
+| `spoCompensationRate` | UnitInterval | Portion allocated to SPOs. |
+| `drepWeightFactor` | UnitInterval | Fraction of DRep pool distributed stake-weighted. |
+| `spoWeightFactor` | UnitInterval | Fraction of SPO pool distributed active-stake-weighted. |
+
+**Constraint:**  
+`ccCompensationRate + drepCompensationRate + spoCompensationRate ≤ MAX_COMP_RATE`  
+where `MAX_COMP_RATE` is hard-coded (e.g., 0.25).
```

Comment:
probably not good to hard-code
best to have everything as an adjustable protocol parameter